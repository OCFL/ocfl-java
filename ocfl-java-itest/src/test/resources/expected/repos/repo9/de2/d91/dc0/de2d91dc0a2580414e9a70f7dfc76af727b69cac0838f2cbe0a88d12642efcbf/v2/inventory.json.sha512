6cd3782bf092c27c2d8e6d719dbdb7f91d21ab6b07718beae3a1dc53e4df5ab09db69734f7214ed3173ff2a07a5813f910082cc35ef46483451a91723f92b4e8  inventory.json
