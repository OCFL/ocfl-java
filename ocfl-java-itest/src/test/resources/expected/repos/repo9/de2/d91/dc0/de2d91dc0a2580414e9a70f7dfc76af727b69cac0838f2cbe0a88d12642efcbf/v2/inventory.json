{
  "id" : "o3",
  "type" : "https://ocfl.io/1.1/spec/#inventory",
  "digestAlgorithm" : "sha512",
  "head" : "v2",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : {
    "70ffe50550ae07cd0fc154cc1cd3a47b71499b5f67921b52219750441791981fb36476cd478440601bc26da16b28c8a2be4478b36091f2615ac94a575581902c" : [ "v1/content/file2" ],
    "79c994f97612eb4ee6a3cb1fbbb45278da184ea73bfb483274bb783f0bce6a7bf8dd8cb0d4fc0eb2b065ebd28b2959b59d9a489929edf9ea7db4dcda8a09a76f" : [ "v1/content/file3" ]
  },
  "versions" : {
    "v1" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "commit message",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "70ffe50550ae07cd0fc154cc1cd3a47b71499b5f67921b52219750441791981fb36476cd478440601bc26da16b28c8a2be4478b36091f2615ac94a575581902c" : [ "file2" ],
        "79c994f97612eb4ee6a3cb1fbbb45278da184ea73bfb483274bb783f0bce6a7bf8dd8cb0d4fc0eb2b065ebd28b2959b59d9a489929edf9ea7db4dcda8a09a76f" : [ "file3" ]
      }
    },
    "v2" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "2",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "70ffe50550ae07cd0fc154cc1cd3a47b71499b5f67921b52219750441791981fb36476cd478440601bc26da16b28c8a2be4478b36091f2615ac94a575581902c" : [ "file2" ]
      }
    }
  }
}